app/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,9 @@
     <uses-permission android:name="android.permission.CAMERA" />
 
     <uses-feature android:name="android.hardware.camera.ar" />
+    <uses-feature
+        android:name="android.hardware.camera"
+        android:required="false" />
```

Comment:
Was this change required to get the tests running?

Reply:
It is not required right now, removing it.